.gitlab-ci.yml

```diff
@@ -159,6 +159,7 @@ check-transaction-versions:
     - job:                         test-linux-stable
       artifacts:                   true
   before_script:
+    - apt-get -y update; apt-get -y install jq
```

Comment:
it's a bad, bad security practice to update the pkg manager within a container,
it's also bad to install something every time the job runs,
why not just add jq to the [CI image](https://github.com/paritytech/scripts/blob/master/dockerfiles/ci-linux/Dockerfile#L26)?